src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error());
+  }, 3000);
+});
+
+function promiseHandler(className, message) {
+  document.body.insertAdjacentHTML('beforeend', `
+    <div class=${className}>
```

Comment:
Could you use `const messageElement = document.createElement('div');` approach instead of raw text?
And add the class name and text node to the messageElement.